src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl {

```

Comment:
Class FileWriterImpl is not implementing any interface. Classes are more flexible and testable when they implement interfaces.

Reply:
I'm not sure which interface could it implements. Should I create custom one ?